frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -62,6 +61,10 @@
   }
 }
 
+.monitoring-dashboards__legend-wrap {
+  overflow-x: auto;
+}
+
```

Comment:
Looks great @kyoto! 
I did see on Chrome the legend svg needed a `max-height` to position the horizontal scrollbar at the bottom correctly.  And a small padding to prevent a clip at mobile. 

```
.monitoring-dashboards__legend-wrap {
  overflow-x: auto;
  padding-top: 1px;
  svg {
    max-height: 50px; // Required for Chrome. legend-wrap - scrollbar height
  }
}
```

Reply:
I tested across the major browsers and it works great!

Reply:
👍 Thanks @sg00dwin! I added those CSS fixes.